cpp/dolfinx/fem/dofmapbuilder.cpp

```diff
@@ -136,147 +136,223 @@ reorder_owned(const std::vector<dofmap_t>& dofmaps, std::int32_t owned_size,
 }
 
 //-----------------------------------------------------------------------------
-
 /// Build a simple dofmap from ElementDofmap based on mesh entity
 /// indices (local and global)
 ///
 /// @param [in] mesh The mesh to build the dofmap on
 /// @param [in] topology The mesh topology
 /// @param [in] element_dof_layout The layout of dofs on each cell type
-/// @return Returns: * dofmap for first element type [0] (local to the process)
+/// @return Returns: * dofmaps for each cell type (local to the process) as
+/// (width, [dofs])
 ///                  * local-to-global map for each local dof
```

Comment:
The last two bullet points are ambiguous. Need to be very precise of 'local' and 'global' - overloaded phrases in the context of dofmaps.